scylla.py

```diff
@@ -143,6 +143,12 @@ def do_deploy_button_callback(self):
             self.log(msg)
 
         self.log("=== Step 2: Copy scylla-package.tar.gz ===")
+        # Prepare the SSH authorized key firstly
+        for info in self.nodes:
+            ip = info[0]
+            cmd = f"ssh-copy-id -i {ssh_key} -o StrictHostKeyChecking=no {ssh_user}@{ip}"
+            self.run_cmd(cmd)
+
```

Comment:
This will require user to input password. We can not assume that user starts the gui app from cmd line and can input password. Currently, we assume the user can use the ssh key provided in the gui to login.